pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2153,6 +2153,14 @@ public class ServiceConfiguration implements PulsarConfiguration {
     )
     private double loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold = 4;
 
+    @FieldContext(
+            dynamic = true,
+            category = CATEGORY_LOAD_BALANCER,
+            doc = "For each uniform balanced unload, the maximum number of bundles that can be unloaded."
+                    + " The default value is -1, which means no limit"
+    )
+    private int maxUnloadBundleNumPerShedding = -1;
```

Comment:
It seems maxUnloadBundleNumPerShedding=0 also means no limit, right?

Reply:
0 means unload cannot be performed. @aloyszhang

Reply:
Got it